Training/Scenes/Repository/RepositoryViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  RepositoryViewController.swift
+//  Training
+//
+//  Created by Ong Yue Huei on 09/07/2021.
+//
+
+import UIKit
+import Nuke
+
+final class RepositoryViewController: UIViewController{
+    struct Dependency {
+        var getRepoUseCase: GetRepoUseCase = GetRepoDefaultUseCase()
+    }
+    
+    @IBOutlet private var actorImage: UIImageView!{
+        didSet {
+            actorImage.layer.cornerRadius = actorImage.frame.size.width * 0.5
+        }
+    }
+    @IBOutlet private var eventBackground: UIView!{
```

Comment:
[nits]
今後はスペースを空けるようにしてください🙇‍♂️
ca_iosではLintで勝手に修正されるようにしています🙆‍♂️

```suggestion
    @IBOutlet private var eventBackground: UIView! {
```

Reply:
承知しました！